apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -205,6 +209,17 @@ export default connect(
       }
     }
 
+    displayOutputToConsole() {
+      if (this.props.logOutput.size > 0) {
+        return this.props.logOutput.map((output, i) => {
+          if (typeof output === 'object') {
+            output = output.toJS();
```

Comment:
I'm not sure this will always be present. In Game Lab, if I `console.log(camera)` I get the following error:

![image](https://user-images.githubusercontent.com/413693/57413275-6685cd00-71a8-11e9-9fea-2e88397dd046.png)

Reply:
Looking into it! Thanks for catching that.